.github/workflows/ci.yml

```
@@ -111,6 +111,8 @@ jobs:
         with:
           version: ${{ matrix.version }}
           arch: ${{ matrix.arch }}
+      - run: julia -e 'using Pkg; Pkg.Registry.add(url="https://github.com/DLR-AMR/T8code.jl", rev="bump-to-t8code-2.0.0")'
+        shell: bash
       - run: julia -e 'using InteractiveUtils; versioninfo(verbose=true)'
       - uses: julia-actions/cache@v1
       - uses: julia-actions/julia-buildpkg@v1
```

```suggestion
      - run: julia -e 'using InteractiveUtils; versioninfo(verbose=true)'
      - uses: julia-actions/cache@v1
      - name: Configure new T8code.jl version environment
        shell: julia --project="." --color=yes {0}
        run: |
          using Pkg
          Pkg.add(PackageSpec(url="https://github.com/DLR-AMR/T8code.jl", rev="bump-to-t8code-2.0.0"))
          Pkg.instantiate()
```
Hm, that doesn't seem to work either. Does this suggestion work (i.e. deleting the`julia-buildpkg` step)? I've seen something like this working in this PR https://github.com/JuliaIO/VideoIO.jl/pull/418/files.

---

This seems to [work now](https://github.com/trixi-framework/Trixi.jl/actions/runs/8848556591/job/24298616080?pr=1918#step:7:310) :+1: